packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.5+3
+
+* Migrating styleFrom API to new version.
```

Comment:
We also usually want to be specific in changelogs; "new version" doesn't provide enough information for the reader to know what exactly is being changed. We should also make it clear that only examples are being changed. How about:

```
* Migrates `styleFrom` usage in examples off of deprecated `primary` and `onPrimary` parameters.
```

Reply:
I agree with you! I changed it to your suggestion. Thank you.